UnitsNet/CustomCode/Wrappers/ReferencePressure.cs

```diff
@@ -25,6 +25,21 @@ namespace UnitsNet.CustomCode.Wrappers
     /// </summary>
     public struct ReferencePressure
     {
+        /// <summary>
+        ///     Represents the pressure at which _pressure is referenced (1 atm default)
+        /// </summary>
+        public static Pressure AtmosphericPressure { get; set; }
+
+        private static readonly Pressure DefaultAtmosphericPressure = new Pressure(1, PressureUnit.Atmosphere);
+
+        /// <summary>
+        ///     Gets a list of <see cref="PressureReference" /> options: <see cref="PressureReference.Gauge" />,
+        ///     <see cref="PressureReference.Absolute" />, and <see cref="PressureReference.Vacuum" />
+        /// </summary>
+        public static List<PressureReference> References { get; } =
```

Comment:
This should perhaps be `IReadOnlyList<PressureReference>` or similar. At the very least an array, to indicate people shouldn't try to manipulate it.

Reply:
Apparently not compatible with .NET Framework 4.0. submitting a PR for an update to .NET Framework 4.8 first.